app/pandas.ipynb

```diff
@@ -471,9 +622,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# make all words in uppercase"
-   ],
+   "source": "s_uppercase = s.str.upper()",
```

Comment:
You have correctly converted the Series to uppercase. Ensure you print the result to verify the output.